action.yaml

```diff
@@ -1,9 +1,9 @@
 name: flutter-version-number
-description: 'Reads the version number of the pubspec.yaml'
+description: "Reads the version number of the pubspec.yaml"
 author: NiklasLehnfeld
 branding:
-  icon: 'archive'
```

Comment:
Changes Like This are Not necessary. It depends on the configuration of the IDE and on personal preferences. I prefer Single quotes, so please Leave it Like this.